mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -113,14 +117,18 @@ def get_signed_conformity_scores(
 
     def get_estimation_distribution(
         self,
+        X: ArrayLike,
         y_pred: ArrayLike,
-        conformity_scores: ArrayLike,
+        values: ArrayLike,
     ) -> NDArray:
         """
         Compute samples of the estimation distribution from the predicted
-        values and the conformity scores, from the following formula:
+        targets and ``values``, from the following formula:
```

Comment:
UPD: according to the comments on the `conformity_score.py` file, this description must be updated.